src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -89,6 +122,16 @@ function CardSection() {
                 )}
                 {isEmptyObject(defaultCard?.accountData) && <CardSectionDataEmpty />}
             </View>
+            {billingStatus?.isRetryAvailable !== undefined && (
```

Comment:
This button + condition is also against the `Add Payment Card` button. Only one of them should show up, so should we've added this to `CardSectionActions`?

Reply:
CardSectionActions renders ThreeDotsMenu so these are separate things as Retry payment button will be displayed below. I believe that it's fine the way it is